easybuild/easyconfigs/i/IMa2p/IMa2p-20151123-foss-2016a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name ='IMa2p'
+version = '20151123'
+commit = '66e7ac7bc5e4c5bbadbd7de943465afeb4641770'
+
+homepage = 'https://github.com/arunsethuraman/ima2p'
+description = """
+IMa2p is a parallel implementation of IMa2, using OpenMPI-C++ - a Bayesian MCMC 
+based method for inferring population demography under the IM (Isolation with 
+Migration) model. http://dx.doi.org/10.1111/1755-0998.12437
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/arunsethuraman/ima2p/archive/']
+sources = ['%s.zip' % commit]
+
+# If CXX is already set the compiler will not be changed to mpic++ by 
+# the configure. We therefore set it to mpic++.
+preconfigopts = 'chmod +x configure && export CXX=mpic++ && '
```

Comment:
another option is to use `toolchainopts = {'usempi': True}` which will make EasyBuild set `$CXX` to the MPI C++ compiler wrapper (+ more, like defining `$OMPI_CXX`)